src/components/LandingPage/Header.tsx

```diff
@@ -145,36 +224,284 @@ const NavBar = ({}) => {
+                        </SubItem>
+                        <SubItem to={links.landing_worker_compensation}>
+                            Worker Compensation
+                        </SubItem>
+                        <SubItem as="a" disabled={true}>
+                            DAOs
+                        </SubItem>
+                        {/* tslint:disable-next-line:jsx-no-lambda */}
+                        <SubItem
+                            as="a"
+                            onClick={(event) => handleNavItemClick('', event)}
+                            href={links.landing_guidebook}
+                            target="_blank"
+                        >
+                            Guidebook
+                        </SubItem>
+                    </SubMenu>
+                </MenuItem>
+                {/* tslint:disable-next-line:jsx-no-lambda */}
+                <MenuItem
+                    onClick={(event) => handleNavItemClick('contribute', event)}
+                >
+                    Contribute
+                    <SubMenu show={navState.contribute}>
+                        <SubItem as="a" disabled={true}>
+                            Project Boards
+                        </SubItem>
+                        {/* tslint:disable-next-line:jsx-no-lambda */}
+                        <SubItem
+                            as="a"
+                            onClick={(event) => handleNavItemClick('', event)}
+                            href={links.landing_omeneth}
+                            target="_blank"
+                        >
+                            Omen
+                        </SubItem>
+                        {/* tslint:disable-next-line:jsx-no-lambda */}
+                        <SubItem
+                            as="a"
+                            onClick={(event) => handleNavItemClick('', event)}
+                            href={links.landing_dxswapeth}
+                            target="_blank"
+                        >
+                            DXswap
+                        </SubItem>
+                        <Divider />
+                        <SubItem as="a" disabled={true}>
+                            Bounties
+                        </SubItem>
+                        <SubItem to={links.landing_codebase}>Codebase</SubItem>
+                    </SubMenu>
+                </MenuItem>
+                {/* tslint:disable-next-line:jsx-no-lambda */}
+                <MenuItem
+                    onClick={(event) => handleNavItemClick('community', event)}
+                >
+                    Community
+                    <SubMenu show={navState.community}>
+                        {/* tslint:disable-next-line:jsx-no-lambda */}
+                        <SubItem
+                            as="a"
+                            href={links.footer_telegram}
+                            target="_blank"
+                            onClick={(event) => handleNavItemClick('', event)}
```

Comment:
Are these onClick handlers being used for anything?

Reply:
yeah they were but it was hacky solution i fixed it in this pr!